website/addons/box/templates/log_templates.mako

```diff
@@ -1,27 +1,30 @@
 <script type="text/html" id="box_file_added">
 added file
-<a class="overflow log-file-link" data-bind="click: NodeActions.addonFileRedirect">{{ params.fullPath }}</a> to
+<a class="overflow log-file-link" data-bind="click: NodeActions.addonFileRedirect">
+    {{ params.fullPath.replace(/^\//, '') }}</a> to
```

Comment:
Rather than repeating the `.replace` call in multiple places, it think it would make sense to add a `stripLeadingSlash` method to the `Log` model so that you can do:

```
{{ stripLeadingSlash(fullPath) }}
```

in the template.